tests/cover/test_type_lookup.py

```diff
@@ -28,7 +28,7 @@
 # Build a set of all types output by core strategies
 blacklist = [
     'builds', 'iterables', 'permutations', 'random_module', 'randoms',
-    'runner', 'sampled_from', 'streaming',
+    'runner', 'sampled_from', 'streaming', 'fspaths',
```

Comment:
The problem here is that xdist fails due to fspaths returning different types and as a result different tests being created. xdist then complains that one runner has a test which the other one hasn't and stops.